### ISSUE TITLE
Ukrainian Wikimedia points to wrong Wikimedia site

### ISSUE THREAD

**Issue by cd19s** [gnosygnu/xowa#21]
On "Help import/List" the entry for the Ukranian Wikimedia points to the wrong URI.  http://uk.wikimedia.org redirects to the Great Britain page: https://wikimedia.org.uk/wiki/Main_Page.  I believe the desired page is https://ua.wikimedia.org/wiki/%D0%92%D1%96%D0%BA%D1%96%D0%BC%D0%B5%D0%B4%D1%96%D0%B0_%D0%A3%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D0%B0 (sorry, copy and paste blew up the URI with hex representations of the Cyrillic alphabet, but the point is the root of the webpage sits at "ua" and not "uk".  This causes the build to fail (can't find file issue, naturally).  (By the way, all other Ukranian sites do start with "uk", this is an oddity.)

Couple of questions while I'm at this:

1. How do I delete "uk.wikimedia.org" from the list of wikis on the left-hand side of the Xowa page (or any wiki I might choose to delete for that matter)?

2.  Is there a way to manually import "ua.wikimedia.org" while still indexing it to "uk.wikimedia.org" in the wikis list (worrying about the British media wiki later)?

**Comment by gnosygnu** [comment]
Hi! Thanks for the catch! 

You're absolutely correct! I added the wikimedia urls based on the WM dump alias (ukwikimedia). It never occurred to me that they would point ukwikimedia instead to ua.wikimedia.org. I'll fix that for tonight's release

> How do I delete "uk.wikimedia.org" from the list of wikis on the left-hand side of the Xowa page (or any wiki I might choose to delete for that matter)?

XOWA just gets the wikis from reading the /xowa/wiki/ filesystem directory on startup.  You can simply move the directory out of the wiki directory. For example, if your wiki is at C:\xowa\wiki\uk.wikimedia.org, move it to C:\some_directory\uk.wikimedia.org. Or just delete it.

> Is there a way to manually import "ua.wikimedia.org" while still indexing it to "uk.wikimedia.org" in the wikis list (worrying about the British media wiki later)?

You should be able to do it through the following:

* Name the ua.wikimedia.org dump "uk.wikimedia.org.bz2"
* Manually import through home/wiki/Help:Import/Script. Use the "other" option

This should create a wiki called "uk.wikimedia.org". This is what I do for the wikia wikis (as well as the nethack one)

Also, just in case you're interested, I'm working on the Ukrainian wiki update. It should be released next Sunday (I've been pushing it off to work on English Wikipedia first).

Thanks again!

**Comment by cd19s** [comment]
I was able to get the Ukranian mediawiki to build.  Thanks.

**Comment by cd19s** [comment]
Found another one.  "ar.wikimedia.org" is actually for Argentina, not Arabic (it looks like the coding for WikiMedia pages is usually, but not necessarily, the same as other pages in the same language group).  There doesn't seem to be a separate Arabic wikimedia - didn't readily see one in the dump list and a Google search only yielded a child page in MetaWiki.

**Comment by gnosygnu** [comment]
> I was able to get the Ukranian mediawiki to build. Thanks.

Good to hear.

> Found another one. "ar.wikimedia.org" is actually for Argentina, not Arabic

Cool. Thanks for finding another.

Yeah, [arwikimedia](http://dumps.wikimedia.org/arwikimedia/latest/) definitely goes to the [Argentinian wikimedia page](https://ar.wikimedia.org/wiki/Página principal). I really wish Wikimedia could have kept their dump names consistent

At any rate, it's going to be a bit hard to fix for tonight. The code assumes that uk* means Ukrainian and ar* means Arabic. I'll have a fix for next week though.

**Comment by gnosygnu** [comment]
This was fixed in tonight's v2.8.4 release. All the new wikimedia sites are now importable. I list all changes below

Thanks again for reporting this.
```
arwikimedia Argentina not Arabic
cawikimedia: Canada not Catalan
bewikimedia: Belgium not Belarusian
brwikimedia: Brazil not Breton
cowikimedia: Columbia not Corsican
sewikimedia: Swedish no Northern Sami
ukwikimedia: Ukrainian and redirect to ua.wikimedia.org
```

**Comment by cd19s** [comment]
I'm glad I could help.
